### PR TITLE
Kvui: assert kivy is not imported before kvui

### PR DESCRIPTION
**Pull request by qwint**
## What is this fixing or adding?
Because kivy needs `KIVY_DATA_DIR` to be redirected to the appropriate folder in the frozen install, importing kivy directly before kvui can cause crashes on some machines using the frozen install. This adds an assert to make sure this workflow gets caught in dev.

relevant issue: https://discord.com/channels/731205301247803413/1261299992564469780

## How was this tested?
did the same `from kivy.metrics import dp` import manual was doing that was causing the crash, it got caught on source
also ran without changes and the error did not trigger,
did not test on source but i can't imagine it does anything because __debug__ should be false

### CODE REVIEW

**Review by hesto2 — APPROVED** [review]
Nice, seems straightforward

File: kvui.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 import re
 from collections import deque
 
+if __debug__:
+    assert "kivy" not in sys.modules, "kvui should be imported before kivy for frozen compatibility"
```
Comment (beauxq) [inline kvui.py:9]:
```suggestion
assert "kivy" not in sys.modules, "kvui should be imported before kivy for frozen compatibility"
```
The `if __debug__` is not necessary. All `assert` statements are removed in the optimized frozen build.

Comment (qwint) [reply kvui.py:9]: that's what i get for thinking a three liner is simple enough after optimizing the `if "kivy" not in sys.modules:` to just be on the assert

**Comment by beauxq** [comment]
A good test would be to make sure you can start all of the core-included clients.

(I tested `ZillionClient.py` and it started up fine.)

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**